cardano-node/src/Cardano/CLI/Tx.hs

```diff
@@ -320,7 +289,10 @@ nodeSubmitTx
   gentx =
     withRealPBFT gHash (Just nId) mNumCoreNodes genFile nMagic sigThresh delCertFp sKeyFp update ptcl $
       \p@Consensus.ProtocolRealPBFT{} -> do
-        case gentx of
-          ByronTx txid _ -> putStrLn $ sformat ("TxId: "%Crypto.hashHexF) txid
-          _ -> pure ()
-        handleTxSubmission socketFp p topology gentx stdoutTracer
+        _ <- case gentx of
+               ByronTx txid _ -> pure . putTextLn
+                                      $ sformat ("TxId: "%Crypto.hashHexF) txid
+               otherTxType -> left . TransactionTypeNotHandledYet
```

Comment:
@denisshevchenko do we want to fail here? I'm not sure.

Reply:
@Jimbo4350 I'm not sure as well. :-) Probably yes.

Reply:
I agree, was just double checking in case there was a reason why you wanted this to fail silently.